### PR TITLE
Implement and test put-object-tagging api

### PR DESCRIPTION
**Pull request by priyanka25081999**
- Experiment put-object-tagging and add to Experiments folder
- Develop a class for put-object-tagging api
- Test the class against cortx and aws
- Add a template file for object tagset
- Update Makefile

Signed-off-by: Priyanka Salunke <priyanka.s.salunke@seagate.com>

### CODE REVIEW

**Comment by priyanka25081999** [comment]
@rajkumarpatel2602 and @mbcortx PR is ready for review.

File: s3/replication/common/src/s3replicationcommon/s3_put_object_tagging.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        self._state = S3RequestState.PAUSED
+        # XXX Take real pause action
+
+    def resume(self):
+        self._state = S3RequestState.PAUSED
+        # XXX Take real resume action
+
+    def abort(self):
+        self._state = S3RequestState.ABORTED
+        # XXX Take abort pause action
```
Comment (rajkumarpatel2602) [inline s3/replication/common/src/s3replicationcommon/s3_put_object_tagging.py:159]: can be removed.

File: s3/replication/common/src/s3replicationcommon/s3_put_object_tagging.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        self._request_id = request_id
+        self._logger = session.logger
+
+        self._bucket_name = bucket_name
+        self._object_name = object_name
+
+        self._tag_name = tag_name
+        self._tag_value = tag_value
+
+        self.remote_down = False
```
Comment (rajkumarpatel2602) [inline s3/replication/common/src/s3replicationcommon/s3_put_object_tagging.py:47]: s/remote_down/_remote_down/g

File: s3/replication/common/src/s3replicationcommon/s3_put_object_tagging.py (hunk 1 above)
Comment (priyanka25081999) [reply s3/replication/common/src/s3replicationcommon/s3_put_object_tagging.py]: done

File: s3/replication/common/src/s3replicationcommon/s3_put_object_tagging.py (hunk 2 above)
Comment (priyanka25081999) [reply s3/replication/common/src/s3replicationcommon/s3_put_object_tagging.py]: done

**Pull request merged**